app/views/groups/_group_list.html.erb

```diff
@@ -0,0 +1,23 @@
+<div id = "contact_dd_arrow_<%=dom_id(group)%>"><i class="fas fa-caret-right" id ="icon_<%=dom_id(group)%>"></i> Click for group details</div>
+
+<div id = "contact_dd_<%=dom_id(group)%>" class= "mt-3" style = "display: none;">
+
+  <% group.contacts.by_created_date.each do |contact| %>
+
+    <%= render "groups/group_list_entry", group: group, contact: contact %>
+
+  <% end %>
+
+</div>
+
+<script>
+
+  $(document).ready(
+      function(){
+          $("#contact_dd_arrow_<%= dom_id(group) %>").click(function () {
+              $("#contact_dd_<%= dom_id(group) %>").fadeToggle();
+              $("#icon_<%= dom_id(group) %>").toggleClass("fas fa-caret-right fas fa-caret-down");
+          });
+      });
```

Comment:
This definitely works as far as changing the icon goes. For the content fading in/out I usually start with a [Bootstrap collapse](https://getbootstrap.com/docs/4.6/components/collapse/).